packages/openai/src/openai-completion-language-model.ts

```diff
@@ -126,6 +127,14 @@ export class OpenAICompletionLanguageModel implements LanguageModelV2 {
         // model specific settings:
         echo: openaiOptions.echo,
         logit_bias: openaiOptions.logitBias,
+        logprobs:
+          typeof openaiOptions?.logprobs === 'number'
+            ? openaiOptions.logprobs
+            : typeof openaiOptions?.logprobs === 'boolean'
+              ? openaiOptions.logprobs
+                ? 0
+                : undefined
+              : undefined,
```

Comment:
Do we need boolean? it seems to just complicate the logic. Wouldn't number be enough?

alternative:

```ts
openaiOptions?.logprobs === true ? 0 : openaiOptions?.logprobs
```

Reply:
good alternative, updated 🙏

Reply:
ugh my suggestion is actually buggy in the `false` case

Reply:
aha i'll fix it